controllers/fluxdeployer_controller.go

```diff
@@ -49,6 +49,8 @@ type FluxDeployerReconciler struct {
 	RegistryServiceName string
 	RetryInterval       time.Duration
 	DynamicClient       dynamic.Interface
+
+	CertSecretName string
```

Comment:
why do we need this here? If it is to permit passing arbitrary `certs` I should rather be another PR.

Reply:
Talked about this on Slack. Because the OCIRepository requires the certificate otherwise it would need an `insecure` setting.